Fix tutorial 1 (#283)
omichel/webots: docs/guide/tutorial-6-4-wheels-robot.md

You can get more information about the node hierarchy in the [nodes chart diagram](../reference/node-chart.md).

> Note that the solid nodes and device nodes are close but don't match perfectly.

Reviewer: ```suggestion
Note that most of the sensors and actuators are [Solid](../reference/solid.md) and [Device](../reference/device.md) nodes at the same time.
```

> The main structure of a robot model is a tree of solid nodes linked together.

Reviewer: ```suggestion
The main structure of a [Robot](../reference/robot.md) model is a tree of [Solid](../reference/solid.md) nodes linked together.
```

> The root node of this tree should be a robot node.

Reviewer: ```suggestion
The root node of this tree should be a [Robot](../reference/robot.md) node.
```

> The solids are linked together by joint nodes.

Reviewer: ```suggestion
The solids are linked together by [Joint](../reference/joint.md) nodes.
```

> A device node should be the direct child of either a robot node, a solid node or a joint node.

Reviewer: ```suggestion
A [Device](../reference/device.md) node should be the direct child of either a [Robot](../reference/robot.md) node, a [Solid](../reference/solid.md) node or a [Joint](../reference/joint.md) node.
```

> A joint node is used to add one (or two) degree(s) of freedom (DOF), between its parent and its child.

Reviewer: ```suggestion
A [Joint](../reference/joint.md) node is used to add one (or two) degree(s) of freedom (DOF), between its parent and its child.
```

> The direct parent and child of a joint node are both solid nodes.

Reviewer: ```suggestion
The direct parent and child of a [Joint](../reference/joint.md) node are both [Solid](../reference/solid.md) nodes.
```

> The nodes derived from Joint allow to create different kinds of constraints between the linked solid nodes.

Reviewer: ```suggestion
The nodes derived from [Joint](../reference/joint.md) allow creating different kinds of constraints between the linked [Solid](../reference/solid.md) nodes.
```

The most used one in robotics, is the [HingeJoint](../reference/hingejoint.md) allowing to model amongst others rotational motors including wheels.

> A joint node can be monitored or actuated by adding respectively a [PositionSensor](../reference/positionsensor.md) node, or a motor node to its `device` field.

Reviewer: ```suggestion
A [Joint](../reference/joint.md) node can be monitored or actuated by adding respectively a [PositionSensor](../reference/positionsensor.md) node, or a motor node to its `device` field.
```

Having these rules in mind, we can start to design the node hierarchy used to model the robot.

> The first step is to determine which part of the robot should be modeled as a solid node.

Reviewer: ```suggestion
The first step is to determine which part of the robot should be modeled as a [Solid](../reference/solid.md) node.
```

> In our case, the body box is obviously the better choice.

Reviewer: 60: The second step is to determine which [Solid](../reference/solid.md) node is the [Robot](../reference/robot.md) node (the root node).

The [figure](#high-level-representation-of-the-4-wheels-robot) depicts the solid nodes hierarchy of the robot.

> > **Hands on #2**: At the end of the scene tree, add a [Robot](../reference/robot.md) node having four [HingeJoint](../reference/hingejoint.md) nodes having a [Solid](../reference/solid.md) as `endPoint`.

Reviewer: ```suggestion
> **Hands on #2**: At the end of the scene tree, add a [Robot](../reference/robot.md) node having four [HingeJoint](../reference/hingejoint.md) nodes having a [Solid](../reference/solid.md) node as `endPoint`.
```

However, actuating the [RotationalMotor](../reference/rotationalmotor.md) nodes is new.

> The following note explain how to proceed.

Reviewer: Note really...

To program the rotational motors, the first step is to include the API module corresponding to the [RotationalMotor](../reference/rotationalmotor.md) node:

> > ```c

Reviewer: ```suggestion
```c
```

> >  #include <webots/motor.h>

Reviewer: ```suggestion
#include <webots/motor.h>
```

> > ```

Reviewer: ```suggestion
```
```

Then to get the references of the [RotationalMotor](../reference/rotationalmotor.md) nodes:

> > ```c

Reviewer: ```suggestion
```c
```

> >  // initialize motors

Reviewer: Same for consecutive lines

>  wb_motor_set_velocity(wheels[0], speed);

> > ```

Reviewer: ```suggestion
```
```

Same for previous lines.
